.travis.yml

```diff
@@ -22,3 +22,12 @@ before_cache:
 - find $HOME/.sbt -name "*.lock" -type f -delete
 - find $HOME/.ivy2/cache -name "ivydata-*.properties" -type f -delete
 - rm -rf $HOME/.ivy2/local
+
+deploy:
+  provider: script
+  script: sbt releaseEarly
+  skip_cleanup: true
+  on:
+    branch: master
+    # only if HEAD is a merge commit:
+    condition: $(git rev-list --merges 'HEAD^..HEAD')
```

Comment:
What does this condition filter out wrt publishing every push on master?

Reply:
This condition is exactly for that: without it every push to master would trigger release (which is ok if master is a protected branch and is never pushed to directly). But with this condition release will be triggered only on merge commits in master.

Reply:
Added also release on tags: 4260877